### PR TITLE
optimize: fix StyleCheck errors for hippo4j-threadpool-core

### LINKED ISSUE #1229: [TASK] About CheckStyle Errors
There are errors related to CheckStyle. I hope someone can fix them and make pull requests.

  - [ ] hippo4j-threadpool-infra-common @yhc777 
  - [x] hippo4j-threadpool-core @dmego 
  - [x] hippo4j-threadpool-adapter-web
  - [x] hippo4j-threadpool-spring-boot-starter-adapter-web
  - [x] hippo4j-threadpool-config-spring-boot-starter
  - [x] spring-plugin-common
  - [x] hippo4j-threadpool-config-spring-boot-1x-starter
  - [x] spring-boot-1.x-plugin
  - [x] spring-boot-2.x-plugin
  - [x] thread-pool-plugin
  - [x] apollo-plugin
  - [x] hippo4j-agent-bootstrap
  - [x] hippo4j-example-core
  - [x] hippo4j-threadpool-config-nacos-example
  - [x] hippo4j-threadpool-config-apollo-example
  - [x] hippo4j-threadpool-config-zookeeper
  - [x] hippo4j-threadpool-config-consul-example
  - [x] hippo4j-threadpool-config-etcd-example
  - [x] hippo4j-threadpool-config-nacos-spring-boot-1x
  - [x] hippo4j-threadpool-config-apollo-spring-boot-1x-example
  - [x] hippo4j-threadpool-server-spring-boot-starter
  - [x] hippo4j-threadpool-adapter-rabbitmq-example
  - [x] hippo4j-threadpool-adapter-stream-rabbitmq-example
  - [x] hippo4j-threadpool-adapter-stream-rocketmq-example
  - [x] hippo4j-threadpool-adapter-rocketmq-example
  - [x] hippo4j-threadpool-adapter-kafka-example
  - [x] hippo4j-threadpool-server-example
  - [x] hippo4j-threadpool-monitor-elasticsearch-example
  - [x] hippo4j-threadpool-rpc
  - [x] hippo4j-threadpool-server-config
  - [ ] hippo4j-agent-core

If everyone wants to fix those errors, they can change <skip.checkstyle.check>true</skip.checkstyle.check> to false and run mvn package to see more details.

Any pull request related to CheckStyle errors can be associated with this issue, and I will update this document when the PR is merged or find new errors.

### PR DESCRIPTION
**Pull request by dmego**
Fixes #1229

Changes proposed in this pull request:
- fix SytleCheck errors for `hippo4j-threadpool-core`

> Check mailbox configuration when submitting. [Contributor Guide](https://hippo4j.cn/community/contributor-guide)

### CODE REVIEW

File: threadpool/core/src/main/java/cn/hippo4j/core/handler/DynamicThreadPoolBannerHandler.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ private void printBanner() {
             while (padding.length() < STRAP_LINE_SIZE - (bannerVersion.length() + DYNAMIC_THREAD_POOL.length())) {
                 padding.append(" ");
             }
-            System.out.println(AnsiOutput.toString(banner, AnsiColor.GREEN, DYNAMIC_THREAD_POOL, AnsiColor.DEFAULT,
+            log.info(AnsiOutput.toString("\n", banner, AnsiColor.GREEN, DYNAMIC_THREAD_POOL, AnsiColor.DEFAULT,
```
Comment (magestacks) [inline threadpool/core/src/main/java/cn/hippo4j/core/handler/DynamicThreadPoolBannerHandler.java:73]: Is the printing format correct after adjustment?

File: threadpool/core/src/main/java/cn/hippo4j/core/plugin/impl/TaskTimeRecordPlugin.java [hunk 2]
```diff
@@ -147,13 +169,8 @@ private Timer getTimerForCurrentThread() {
      * copy from {@link HashMap#tableSizeFor}
      */
     static int tableSizeFor(int cap) {
```
Comment (magestacks) [inline threadpool/core/src/main/java/cn/hippo4j/core/plugin/impl/TaskTimeRecordPlugin.java:171]: Please add unit test for this method

Comment (dmego) [reply threadpool/core/src/main/java/cn/hippo4j/core/plugin/impl/TaskTimeRecordPlugin.java:171]: This is the implementation of the `HashMap#tableSizeFor` method in `jdk 11`. If needed, I will provide additional unit test cases.

File: threadpool/core/src/main/java/cn/hippo4j/core/handler/DynamicThreadPoolBannerHandler.java (hunk 1 above)
Comment (dmego) [reply threadpool/core/src/main/java/cn/hippo4j/core/handler/DynamicThreadPoolBannerHandler.java:73]: Yes, I have conducted tests, and the print format is correct. Just like this: 

image

File: threadpool/core/src/main/java/cn/hippo4j/core/plugin/impl/TaskTimeRecordPlugin.java (hunk 2 above)
Comment (magestacks) [reply threadpool/core/src/main/java/cn/hippo4j/core/plugin/impl/TaskTimeRecordPlugin.java:171]: Need to provide, the unit test needs to write the result comparison of two pieces of code.

**Review by magestacks — APPROVED** [review]

**Pull request merged**